modin/pandas/__init__.py

```diff
@@ -96,6 +96,8 @@
         NamedAgg,
         NA,
         api,
+        tseries,
+        offsets,
```

Comment:
Don't these modules return objects that Modin should wrap in Series or DataFrame?

Reply:
not offsets.  some other stuff in tseries yes